app/assets/src/components/Samples.jsx

```diff
@@ -740,22 +736,61 @@ class Samples extends React.Component {
       </div>
     );
 
+    const project_menu = (
+      <div className='right'>
+        <ul className='project-menu'>
+          <li>
+            { this.state.project ? (
+              this.state.project.public_access ?
+               <span>
+                 <i className="tiny material-icons">lock_open</i> Public Project
+               </span>:
+               <span>
+                 <i className="tiny material-icons">lock</i> Private Project
+               </span>
+            ) : null }
+          </li>
+          <li>
+              { this.state.project ? (
+                this.state.project.total_members ?
+                <span>
+                  <i className="tiny material-icons">people</i> {this.state.project.total_members} Member(s)
```

Comment:
In this PR or the next, please get rid of the "(s)" and check the plural to make it correct. There must be a "pluralize" function out there that does it automatically